wandb/sdk/artifacts/artifact.py

```diff
@@ -330,8 +320,11 @@ def new_draft(self) -> "Artifact":
 
         The artifact returned can be extended or modified and logged as a new version.
 
+        Returns:
```

Comment:
Missing Return description.

Reply:
I will wait to get confirmation from eng team @estellazx on this - I think this returns an Artifact object though as well?